### PR TITLE
Fixed around func

### LINKED ISSUE #15484: rounding.test_numpy_around

### PR DESCRIPTION
**Pull request by NicholusMagak**
This should to fix #15484 which is related to #15292.

### CODE REVIEW

**Comment by NicholusMagak** [comment]
Hi @MahmoudAshraf97. Could you check this PR? I know it has some failing checks and would appreciate your help on those areas of concern. Thank you.

**Comment by NicholusMagak** [comment]
@MahmoudAshraf97 In the event you have some time, could you assist with this PR?

**Comment by NicholusMagak** [comment]
@MahmoudAshraf97 I'm not sure if you saw the previous comments, but, in the event you have some time, your help with this PR would be appreciated.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_rounding.py [hunk 1]
```diff
@@ -213,22 +213,20 @@ def test_numpy_rint(
...
 
+# around
 # around
 @handle_frontend_test(
     fn_tree="numpy.around",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("numeric"),
+        available_dtypes=helpers.get_dtypes(),
     ),
-    decimals=st.integers(min_value=0, max_value=5),
```
Comment (MahmoudAshraf97) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_rounding.py:222]: we should test for all arguments

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_rounding.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # global
-from hypothesis import strategies as st
+# from hypothesis import strategies as st
```
Comment (MahmoudAshraf97) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_rounding.py:2]: please revert this

File: ivy/functional/frontends/numpy/mathematical_functions/rounding.py [hunk 3]
```diff
@@ -113,7 +113,12 @@ def _rint(
...
 @to_ivy_arrays_and_back
 @from_zero_dim_arrays_to_scalar
-def around(a, decimals=0, out=None):
+def _around(
+    a,
+    decimals=0,
+    out=None,
+):
     if ivy.shape(a) == ():
         a = ivy.expand_dims(a, axis=0)
```
Comment (MahmoudAshraf97) [inline ivy/functional/frontends/numpy/mathematical_functions/rounding.py:122]: why is this necessary?

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/numpy/mathematical_functions/rounding.py (hunk 3 above)
Comment (NicholusMagak) [reply ivy/functional/frontends/numpy/mathematical_functions/rounding.py:122]: It was to handle the potential event of a zero dimensional that would have been an input 'a' and change it to 1d. This was done by the person who previously tried to implement this function.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
